### PR TITLE
Metadata feature

### PR DESCRIPTION
**Pull request by vmchokk**
Adds metadata feature

### CODE REVIEW

File: docs/epayment.v1.yml [hunk 1]
```diff
@@ -308,6 +308,22 @@ components:
...
         - NOK
       description: Currency code as defined in ISO 4217. eg NOK for Norwegian kroner.
+    Metadata:
+      type: object
+      description: |-
+        Metadata is a key-value map that can be used to store additional
+        information about the payment. The metadata is not used by Vipps
+        MobilePay, but is passed through in the `GetPaymentResponse` object.
+        Key length is limited to 100 characters, and value length is limited to 500 characters.
+        Max capacity is 5 key-value pairs.
```
Comment (khenrichsen) [inline docs/epayment.v1.yml:318]: do we allow any other values than string?

Comment (OFollan) [reply docs/epayment.v1.yml:318]: Nopes :) Maybe add some text here to explain that 👍

Comment (OFollan) [reply docs/epayment.v1.yml:318]: Or is that the `additionalProperties: type: string` stuff 🤔 ?

File: docs/epayment.v1.yml [hunk 2]
```diff
@@ -308,6 +308,22 @@ components:
...
+      additionalProperties:
+        type: string
+      maxProperties: 5
+      example:
+        key1: value1
+        key2: value2
+        key3: value3
```
Comment (OFollan) [inline docs/epayment.v1.yml:325]:
```suggestion
        key1: "value1"
        key2: "value2"
        key3: "value3"
```

**Review by OFollan — APPROVED** [review]

**Pull request merged**